default.yaml

```diff
@@ -117,3 +117,5 @@ lnds:
 onchainDustAmount: 5000
 # TODO
 # carrierRegexFilter: ""
+
+shareMemoWithPayee: true
```

Comment:
I don't think we want a global level settings for that value. It doesn't make sense to switch this on/off really.

eventually we could have a way, within the query. to choose whether one user can select to share the memo or not. 

for now I would share the memo and see what feedback we have once this is merged.

Reply:
Fixed